srcpkgs/nagios/template

```diff
@@ -0,0 +1,57 @@
+# Template file for 'nagios'
+pkgname=nagios
+version=4.4.6
+revision=1
+build_style=gnu-configure
+
+_nagios_user="_nagios"
+_nagios_group="_nagios"
+_instdir="/usr/share/nagios"
+_bindir="/usr/bin"
+_vardir="/var/nagios"
+_confdir="/etc/nagios"
+_httpdconfdir="/etc/webapps/nagios/"
+_checkresultdir="/var/nagios/spool/checkresults"
+_cgibindir="/usr/libexec/nagios/cgi"
+
+configure_args="
+ --with-nagios-user=$_nagios_user
+ --with-nagios-group=$_nagios_group
+ --prefix=$_instdir
+ --bindir=$_bindir
+ --localstatedir=$_vardir
+ --sysconfdir=$_confdir
+ --with-httpd-conf=$_httpdconfdir
+ --with-checkresultdir=$_checkresultdir
+ --with-cgibindir=$_cgibindir
+ --enable-embedded-perl"
+make_build_args="all"
+make_install_args="install install-config"
+
+hostmakedepends="perl unzip"
+make_dirs="
+$_confdir 0775 _nagios _nagios
+$_vardir 0755 _nagios _nagios
+$_vardir/rw 0755 _nagios _nagios
+$_vardir/spool 0755 _nagios _nagios
+$_vardir/spool/checkresults 0755 _nagios _nagios"
```

Comment:
Can `nagios` `mkdir` its subdir itself?
I think the permission should be 770 as the same with nagios default.

Reply:
I will update the permissions.

Nagios creates some of these directories during the installation process (though not all of them), however the void package hooks delete the directories since they are empty.

Reply:
I meant can nagios create those directories upon startup.
The current `make_dir` trigger doesn't delete the directories (upon removal) in reverse orders so ...

Reply:
I looked through the configuration parameters and the CLI args to `nagios` and cannot see any way for it to create the directories it needs.

It would be possible to move this directory creation logic into the runit `run` script.  It would be a couple of `mkdir` commands followed by `chown` commands.  Would this be preferred? I'm personally ok with any methodology for creating these directories.